ping_app/ping_service.py

```diff
@@ -83,6 +83,12 @@ async def get_current_zones_status() -> str:
         zone_statuses = [notifier.get_current_state(zone) for zone in zones]
         return '\n'.join(zone_statuses)
 
+    @staticmethod
+    async def get_current_zones_status_short() -> str:
+        zones = await host_crud_service.get_all_zones()
+        zone_statuses = [notifier.get_current_state_short(zone) for zone in zones]
```

Comment:
I'm not sure but would this work?
```suggestion
        zone_statuses = [notifier.get_current_state_short(zone) for zone in await host_crud_service.get_all_zones()]
```

Reply:
this will work, but worse readability imho